portal/flatpak-portal.c

```diff
@@ -2716,7 +2716,8 @@ read_variant (GInputStream *in,
               GError **error)
 {
   guint32 size;
-  guchar *data;
+  g_autofree guchar *data = NULL;
+  guchar *data_owned;
```

Comment:
Seems backwards that the "owned" variable is not actually owning (and thus using g_autofree).